scripts/blog_template.py

```diff
@@ -1,12 +1,46 @@
 import os
 import sys
 import pandas as pd
+import argparse
 from jinja2 import Environment, FileSystemLoader
 
 script_dir = os.path.dirname(__file__)
 
+# a mapping of column names from the MS Form to names
+# used in the jinja template
+col_dict = {
+    "ID": "ID",
+    "Start time": "Start time",
+    "Completion time": "Completion time",
```

Comment:
don't these values need to follow the template without space?

Reply:
They aren't actually needed by jinja (they're never used in the template file) so we get away with keeping them the same.

Reply:
I guessed, but thought it best to double check.